### PR TITLE
Change references from test-integration-cli to test-integration

### PR DESCRIPTION
**Pull request by asottile**
**- What I did**
manual human `sed`

**- How I did it**

**- How to verify it**
:eyes:

**- Description for the changelog**
Update testing documentation to remove deprecated `integration-test-cli` target mentions.

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: docs/contributing/test.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ testing:
 | ---------------------- | ---------------------------------------------- |
 | `test`                 | Run the unit, integration, and docker-py tests |
 | `test-unit`            | Run just the unit tests                        |
-| `test-integration-cli` | Run the integration tests for the CLI          |
+| `test-integration`     | Run the integration tests for the CLI          |
```
Comment (AkihiroSuda) [inline docs/contributing/test.md]: You can remove "for the CLI"

Comment (asottile) [reply docs/contributing/test.md]: done!

**Review by AkihiroSuda — APPROVED** [review]
lgtm

cc @dnephin @vdemeester

**Review by dnephin — APPROVED** [review]
LGTM

**Pull request merged**